includes/admin/class-wcs-admin-post-types.php

```diff
@@ -1811,9 +1811,7 @@ public function orders_table_query_clauses( $pieces, $query, $args ) {
 
 		$query_order = strtoupper( $args['order'] );
 
-		// fields and order are identical in both cases
-		$pieces['fields'] .= ', COALESCE(lp.last_payment, orders.date_created_gmt, 0) as lp';
```

Comment:
I've removed the changing of the fields param because it isn't necessary to add additional `SELECT` fields to a query thats just fetching results to display in the list table.

We were only selecting the `COALESCE( ... )` in order to order by that result and I can do the `COALESCE` in the `orderby` anyway.